index.html

```diff
@@ -323,6 +323,140 @@
             is not yet established or being established.</li>
         </ul>
       </p>
+      <section>
+        <h4>
+          Initiate remote playback
+        </h4>
+        <p>
+          When the <code><dfn for="RemotePlayback">connect</dfn>()</code>
```

Comment:
Hmm, now that `connect()` is defined, can you remove the comment in the interface definition?